lib/widgets/message_list.dart

```diff
@@ -1381,7 +1381,9 @@ class MessageWithPossibleSender extends StatelessWidget {
                 ])),
               SizedBox(width: 16,
                 child: message.flags.contains(MessageFlag.starred)
-                  ? Icon(ZulipIcons.star_filled, size: 16, color: designVariables.star)
+                  ? Transform.translate(
+                    offset: const Offset(-2,0),
```

Comment:
When the UI is showing in a right-to-left (RTL) language, this will move the star in the wrong direction:

<img width="581" alt="image" src="https://github.com/user-attachments/assets/d8861b84-57be-4d2c-bea4-fc54dabdbd56" />

I don't see an `OffsetDirectional` (on the pattern of [`AlignmentDirectional`](https://api.flutter.dev/flutter/painting/AlignmentDirectional-class.html), [`EdgeInsetsDirectional`](https://api.flutter.dev/flutter/painting/EdgeInsetsDirectional-class.html), etc.), so I think the solution is to write a short conditional on the value of [`Directionality.of(context)`](https://api.flutter.dev/flutter/widgets/Directionality/of.html).